tools/@aws-cdk/ubergen/bin/ubergen.ts

```diff
@@ -3,9 +3,12 @@ import * as path from 'path';
 import * as process from 'process';
 import cfn2ts from '@aws-cdk/cfn2ts';
 import * as cfnspec from '@aws-cdk/cfnspec';
+import * as monocdk from '@monocdk-experiment/rewrite-imports';
```

Comment:
Weeellll...

At this point I'd rather do it like this:

* Put a parameterized version of the rewriter into `aws-cdk-migration` (have it take the monopackage name as an argument)
* Have `aws-cdk-migration` call that function in itself with `aws-cdk-lib`.
* Have `@monocdk-experiment/rewrite-imports` depend on `aws-cdk-migration`, and have it pass `monocdk` into that parameter.
* Have `ubergen` depend on `aws-cdk-migration` and have it pass the right monopackage name.

Let's dedupe this crap properly!